wlutil/build.py

```diff
@@ -43,7 +43,16 @@ def handleHostInit(config):
            raise ValueError("host-init script " + str(config['host-init']) + " not found.")
 
        run([config['host-init'].path] + config['host-init'].args, cwd=config['workdir'])
- 
+
+def handleBinPost(config):
```

Comment:
Since marshal can be run from anywhere these days, and kernel sources can move around and stuff, I think it'd be a good idea to tell the post-bin script where the kernel binary and source are. At a minimum, it will make the scripts easier to write, but it will also make the system less brittle. I'm thinking we could add two environment variables for the script to use:
FIREMARSHAL_LINUX_SRC
FIREMARSHAL_LINUX_BIN

The SRC one will point the source actually used to build the bin which could be the default one, or in a board directory, or in the workload's workdir directory (it's whatever is in config['linux-src']).

we'd also need to add these to the docs.